### PR TITLE
Urgent small fixes in the Dockerfile and Compose configs

### PR DESCRIPTION
**Pull request by kkovaletp**
Freeze UID and GID for the `photoview` user inside the image; added alternative permission configs description to the Readme and Makefile (as a comment); switched the default tag in the compose from `2` to `master`, as this is the most common user misunderstanding according to the issues and Discord questions.

The tag change in the compose files is needed, as the `2` image is too old, and contains a lot of fixed bugs, which users keep reporting. It is not obvious for them to switch the tag. Also, I don't believe the release is happening soon. Anyway, we could change the tag back right before the release.

### CODE REVIEW

**Comment by viktorstrate** [comment]
Regarding changing the tag to master. Why don't we make more releases, what's holding us back right now?

File: Dockerfile
Comment (viktorstrate) [inline Dockerfile:66]: What implications does this have on existing setups. Could this break existing setups such that Photoview no longer has access to the files?

**Comment by kkovaletp** [comment]
@viktorstrate,

> Regarding changing the tag to master. Why don't we make more releases, what's holding us back right now?

Right now we have at least #797 waiting for the finalization and merge. As far as I understand from @jordy2254 comments, a small code change is required, but the original contributor is not responsive, so I asked @jordy2254 to step in and make the needed change, following by merge of the PR. Waiting for this to happen.

While this is the only PR left not merged from the original release plan, we have now 1 more PR #965, fixing an annoying UI issue. The fix is implemented and tested manually, but I cannot finish unit tests development for the module, as they keep failing no matter what I try from AI suggestions. I need some help from a developer to debug and fix those unit tests before the PR can be reviewed and merged. I asked @jordy2254 for help and waiting for his free time to deep dive into the frontend and find out the way to fix them...

Basically, the main issue with the release is that our project needs more experienced developers with the maintainer role, so that we don't have a bottleneck with PR review and consultancy. Saying this, I completely understand that we all here work for fun in our free time and don't get any salary, so no one is obligated to spend a minimal amount of hours per week on the project. But still: the issue is not solved. Some time ago I posted the next invite on my [Linkedin page](https://www.linkedin.com/feed/update/urn:li:activity:7204541422807560192/), but didn't get any responses and don't see volunteers in the project. You might want to repost it for your network or post something similar to your friends) I don't know how else I can help to solve this problem...

File: Dockerfile
Comment (kkovaletp) [reply Dockerfile:66]: Right now the image is created with the user and group, having UID=999 and GID=999, but as the command gets the 1st free ID, there is no guarantee that in the future after base image update these IDs won't change. That is why even while users see the IDs inside the container, they can't rely on them when managing FS permissions on the host. 

With this change, the IDs would be frozen/hardcoded, but still the same as currently used. This means, that those, who already configured their permissions on the host with the current IDs, will not see any difference, while all others will be able to do the same and rely on the IDs. If in the future at least 1 of the IDs is locked and not available at the build time, the image build will fail and we'll get the explicit error about this issue instead of automatically using some other ID and breaking users' setup.

**Comment by viktorstrate** [comment]
But is the issue blocking the release, why are we waiting for it to be merged?

**Comment by kkovaletp** [comment]
@viktorstrate, 

#797 is an improvement. Not a bug fix. So, it is not blocking the release.
#965 is a bug fix for an annoying user-visible issue, but it is not a blocker and the product can be used "as is".

I'll understand if you'd like to push the release without these 2 PRs being included. However, if we can make them ready and merge before the release, I think that the final release build would be much better.

Please let me know your decision, so that I can update this PR with the correct tag so that it is included in the release anyway.

**Comment by viktorstrate** [comment]
I think it's better to have smaller and more frequent releases, rather than recommending users to use the master tag ☺️

**Comment by kkovaletp** [comment]
@viktorstrate, ok. I've reverted back the tag-related changes in the compose files.
Please approve (I've enabled auto-merge for this PR, so once all jobs are passed and there is approval, it will be merged automatically).

Regarding the release, if you think that the product is ready for the release (after merging this PR), feel free to perform the needed actions, as an owner and leader of the project)

**Review by viktorstrate — APPROVED** [review]

**Pull request merged**
